src/families/polkadot/preload.js

```diff
@@ -56,11 +58,15 @@ function fromHydratePreloadData(data: mixed): PolkadotPreloadData {
         bondingDuration: Number(bondingDuration) || 28,
       };
     }
+
+    // $FlowFixMe
+    minimumBondBalance = BigNumber(data.minimumBondBalance);
```

Comment:
i think you should do 

```
BigNumber(data.minimumBondBalance || 0)
```

because there is a slight possibility that you will first get hydrate() WITHOUT the data when coming on old Live.